pkg/ui/README.md

```diff
@@ -83,6 +83,14 @@ dependencies properly, so a top-level `make build` will rebuild exactly the
 necessary DLLs.) DLLs also make the Webpack configuration rather complicated.
 Still, the tradeoff seems well worth it.
 
+## CCL Build
+
+In CCL builds, code in `pkg/ui/ccl/src` overrides code in `pkg/ui/src` at build
+time, via an import resolution rule. E.g. if a file imports
+`src/views/shared/components/licenseType`, it'll resolve to
+`pkg/ui/src/views/shared/components/licenseType` in an OSS build, and
+`pkg/ui/ccl/src/views/shared/components/licenseType` in a CCL build.
+
```

Comment:
Probably worth adding a note here about the explicit override mechanism.  In CCL code, when a component wants to import the OSS version for the purpose of an expired-license fallback, it should write the same import prefixed with `oss`.  As a matter of style this should only be done in a CCL file importing the exact same file from OSS, and in no other cases.